daphne/src/lib.rs

```diff
@@ -526,4 +526,5 @@ mod messages_test;
 pub mod roles;
 #[cfg(test)]
 mod roles_test;
+pub mod testing;
```

Comment:
The intention for this new module is that it will expose a number of utilities for folks to test Daphne with. It would not make sense to put it behind `test`, since it would be only available in this crate's tests.